expression/distsql_builtin.go

```diff
@@ -1038,6 +1038,19 @@ func newDistSQLFunctionBySig(sc *stmtctx.StatementContext, sigCode tipb.ScalarFu
 	}, nil
 }
 
+// PBToExprs converts pb structures to expressions.
+func PBToExprs(pbExprs []*tipb.Expr, fieldTps []*types.FieldType, sc *stmtctx.StatementContext) ([]Expression, error) {
```

Comment:
IMO, there is no need to define this function.

Reply:
Why? The function has 3 usage.

Reply:
OK